debian/noembed/nginx/sites/participate.whatwg.org.conf

```diff
@@ -5,7 +5,7 @@ server {
 
   add_header Strict-Transport-Security "max-age=63072000; includeSubDomains; preload";
   add_header X-Content-Type-Options nosniff;
-  add_header X-Frame-Options DENY;
+  add_header X-Frame-Options deny always;
```

Comment:
Where does always come from? https://tools.ietf.org/html/rfc7034 doesn't seem to list it.

Reply:
It's for nginx, not sent on the wire.

Reply:
Should I could quote "deny" to make that a bit more obvious?

Reply:
Why are the others not always? Maybe add a comment...

Reply:
I think the others should be always too then.

Reply:
Always always it is.